e2e_tests/test_workspace_templates.py

```diff
@@ -33,4 +33,4 @@ async def test_get_workspace_templates(template_name, verify) -> None:
 async def test_get_workspace_template(template_name, verify) -> None:
     admin_token = await get_admin_token(verify)
     async with get_template(template_name, strings.API_WORKSPACE_TEMPLATES, admin_token, verify) as response:
-        assert (response.status_code == status.HTTP_200_OK), f"GET Request for {template_name} creation failed"
+        assert_status(response, [status.HTTP_200_OK], f"Failed to create template: {template_name}")
```

Comment:
does it necessarily apply that the creation has failed?

Reply:
@yuvalyaron what do you mean?

Reply:
if we fail to get the workspace template - we see a message that says that the creation of the template has failed,
I'm asking if there are other options. 
i.e. the creation was ok, but there's a problem with the token, or with the endpoint.
Is it ok to assume that it's the creation that failed?

Reply:
because the test is not creating the template, it just calls GET on it

Reply:
I made a mistake in the error message. it should be:
```suggestion
        assert_status(response, [status.HTTP_200_OK], f"Failed to GET template: {template_name}")
```